Update GITHUB_PAT guidance (#93)
moj-analytical-services/user-guidance: source/documentation/github.md

To generate your PAT for reading private repos in R:

>   1. Click [this link](https://github.com/settings/tokens/new?scopes=repo&description=R:GITHUB_PAT) to take you to right Github settings page. Or else you can navigate there on the Github site by going to **_Settings > Developer Settings > Personal Access Tokens_**

Reviewer: ```suggestion
  1. Navigate to the [GitHub personal access tokens page](https://github.com/settings/tokens/new?scopes=repo&description=R:GITHUB_PAT). You can also access this from any other GitHub page by selecting your user icon, then **Settings**, then **Developer settings**, then **Personal access tokens**.
```

### Using a PAT to authenticate in R/RStudio

> You should store your PAT in a special R file, called .Renviron, in your *home directory* (on AP). This file gets run when you start R, putting the PAT into the system environment variable `GITHUB_PAT`. This is where common R packages (e.g. remotes, devtools and renv) will look for it.

Reviewer: ```suggestion
You should store your PAT in a special R file, called `.Renviron` in your home directory on the Analytical Platform. This file gets run when you start R, putting the PAT into the system environment variable `GITHUB_PAT`. This is where common R packages (for example, remotes, devtools and renv) will look for it.
```
